### DISCUSSION TITLE
Engine 1 inop after MS/FS July 27th performance update

### DISCUSSION THREAD

**Discussion by Mickepilot** [flybywiresim/aircraft#5426] [Q&A] [answered]
Hello folks, maybe somebody can help?
After 4 hrs download (about 40GB, with the new FS performance upgrade), I started the a32nx (Development version) flight
airborne on final RWY 14 LSZH.
What suprised me: ENG 1 Master switch is in OFF position and can not be turned on. So I made a left engine inop landing.
Then I tried a TakeOff RWY 28 LSZH, also here, the ENG 1 Master switch can NOT be turned on. So no TakeOff.
Same with Asobos A320neo.
The Thrustmaster TCA Airbus Edition control panel shows both ENG 1 & 2 switches working fine (off - on - off)
Also FS game settings show correct Master switch bindings with Thrustmaster's buttons 3 & 4.
Even when trying with the mouse, the Master switch 1 can not be switched on.
Anybody have an idea to get the Master switch 1 working again?
Everything (FS, FlyByWire, Thrustmaster) has the latest updates.
Thank's for any help.
And the good thing: the new update gives very much smoother visuals, bravo.

**Comment by tracernz** [comment, marked as answer]
There are some additional functions that need to be bound to the master switches. Unfortunately the update seems to reset some of your bindings. The #support channel on Discord will have the details.

**Reply by Mickepilot** [reply]
Sorry, I have no access to Discord (account problem). Can anybody tell me the details from Discord?
How come that the ENG 2 master switch working perfect, if the bindings is reset by the update?

**Reply by lcornar** [reply]
I'm having the same problem with my TCA quadrant. Engine 2 switch is perfectly working but engine 1 doesn't. I tried to reset the bindings of button 3 (engine 1 switch) but it still doesn't work. When I load in runway I can't do nothing with engine 1 switch (nor with the TCA nor with mouse on cockpit, It seems bugged) but when I load on gate I'm able to use engine 1 switch with mouse but not from TCA.

**Reply by lcornar** [reply]
I found this in discord. My TCA is working perfectly again. Thank you guys from flybywire team! 

![IMG_20210728_020016](https://user-images.githubusercontent.com/79520228/127242572-db356ad3-f1e3-47bf-8118-9bb597044f87.jpg)

**Reply by Mickepilot** [reply]
... YES, this was the fix, I'm back in business, thank's very much for the tip!

**Comment by emsss78** [comment]
Do I need to remap the buttons again or just remove them as mentioned??

**Reply by Mickepilot** [reply]
@emsss78, after applying the above mentioned change, I got ENG 1 running fine, but after landing I could not switch off the engine 1.
After remap of the ENG 1 Master in the FS settings I got it working normally again.

**Reply by emsss78** [reply]
> @emsss78, after applying the above mentioned change, I got ENG 1 running fine, but after landing I could not switch off the engine 1.
> After remap of the ENG 1 Master in the FS settings I got it working normally again.

Hmmm..interesting...thanks for the observation...i'll check it out..i'll do a test flight in a bit and see how it goes..so far no CTD's after SU5
